yt_dlp/extractor/bilibili.py

```diff
@@ -1414,7 +1578,7 @@ def _real_extract(self, url):
                 'title': ('title', {str}),
                 'uploader': ('upper', 'name', {str}),
                 'uploader_id': ('upper', 'mid', {str_or_none}),
-                'timestamp': ('ctime', {int_or_none}),
+                'timestamp': ('ctime', {int_or_none}, {lambda x: x or None}),
```

Comment:
Or just:
```suggestion
                'timestamp': ('ctime', {lambda x: int(x) or None}),
```

Reply:
I like the current version more. It explicitly shows why `int_or_none` is not enough (i.e. we want to filter out 0). `{lambda x: x or None}` can later also easily be replaced with `{truthy_or_none}` or whatever that function will be called.